Gordon360/Static Classes/Names.cs

```diff
@@ -29,13 +29,14 @@ public static class Resource
         public const string CHECKIN = "Info relating to a student's Academic Check-In";
         public const string SHIFT = "A shift that a student has worked";
         public const string CLIFTON_STRENGTHS = "A student's uploaded clifton strengthsfinder results";
-        public const string RECIM = "General/Admin RecIM resource";
+        public const string RECIM = "A general or admin RecIM resource";
         public const string RECIM_ACTIVITY = "A RecIM activity resource";
         public const string RECIM_SERIES = "A RecIM series resource";
         public const string RECIM_MATCH = "A RecIM match resource";
         public const string RECIM_TEAM = "A RecIM team resource";
         public const string RECIM_SPORT = "A RecIM sport resource";
-        public const string RECIM_PARTICIPANT = "RecIM Participating User (FacStaff or Student)";
+        public const string RECIM_PARTICIPANT = "A RecIM Participating User (FacStaff or Student) resource";
+        public const string RECIM_PARTICIPANT_ADMIN = "A RecIM Participating Admin resource";
```

Comment:
```suggestion
        public const string RECIM_PARTICIPANT_ADMIN = "The admin status of a RecIM participating user";
```
Does this make more sense in context of what this resource actually controls?

Reply:
I think so, but I also think there is a possibility that we could have other admin related updates that might need to be made down the line (things like different levels of admin permissions, which came up at some point). Potentially we could make it more specific right now but I could see a near future where it would just be reverted?